app/index.html

```diff
@@ -74,7 +74,38 @@
     </div>
 
     <div class="container">
-    <div ng-view=""></div>
+        <div class="feedbackSAM" ng-show="feedbackSAM">
+            <span class="hideButton glyphicon glyphicon-remove" ng-click="feedbackSAM = false"></span>
+
+            <div id="valenceFeedback" class="feedback">
+                <span class="title">How positive is this song?</span>
+                <div class="elements">
+                    <div class="element" ng-repeat="n in [1,2,3,4,5]">
+                        <label for="valenceFeedback{{n}}">
+                            <img width="40px" height="60px" src="/images/valence/valence{{n}}.png" alt="valence{{n}}" />
+                        </label>
+                        <input id="valenceFeedback{{n}}" name="valenceFeedback" type="radio" value="{{n}}"/>
+                    </div>
+                </div>
+            </div>
+
+            <div id="arousalFeedback" class="feedback">
+                <span class="title">How excited is this song?</span>
+                <div class="elements">
+                    <div class="element" ng-repeat="n in [1,2,3,4,5]">
+                        <label for="arousalFeedback{{n}}">
+                            <img width="40px" height="60px" src="/images/arousal/arousal{{n}}.png" alt="arousal{{n}}" />
+                        </label>
+                        <input id="arousalFeedback{{n}}" name="arousalFeedback" type="radio" value="{{n}}"/>
+                    </div>
+                </div>
+            </div>
+
+            <button type="submit" value="Submit feedback" ng-click="sendClassification()">
```

Comment:
If it's a submit button, it should have a form, and te function should be defined as `ng-submit` on the form (this would allow for enter submission if the form is active). Otherwise, it should be a `type="button"`.